public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/MetricStatEditor.test.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
-import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+import { cleanup, fireEvent, render, screen, act } from '@testing-library/react';
 import { setupMockedDataSource } from '../../__mocks__/CloudWatchDataSource';
 import '@testing-library/jest-dom';
 import { CloudWatchMetricsQuery } from '../../types';
 import userEvent from '@testing-library/user-event';
 import { MetricStatEditor } from '..';
+import selectEvent from 'react-select-event';
```

Comment:
nit: external imports should be at the top. this should be enforced by es-lint and fixed by prettier but currently it's not

Reply:
🤭  indeed, moving it now

Reply:
I have started to use this setting in my vscode:

```
  "typescriptHero.imports.organizeOnSave": true,
```

(if you have the typescriptHero extension). It automatically organize and remove unused imports when you have, quite useful.

Reply:
Neat!